.github/workflows/continuous-integration.yml

```
@@ -0,0 +1,43 @@
+name: continuous-integration
+on: [pull_request]
+jobs:
+  test-runner:
+    runs-on: ubuntu-18.04
+    steps:
+    - uses: actions/checkout@v2
+    - name: Setup node
+      uses: actions/setup-node@v2
+      with:
+        node-version: '12'
+        check-latest: true
+    - name: Install prerequisites
+      run: | 
+       sudo apt-get install g++-4.8 xvfb
+       sudo apt-get install xserver-xorg-dev libxext-dev libxi-dev 
+       sudo apt-get install -y build-essential libxi-dev libglu1-mesa-dev libglew-dev pkg-config libglu1-mesa-dev freeglut3-dev mesa-common-dev libcairo2-dev
```

```suggestion
       sudo apt-get install -y build-essential libxi-dev libglu1-mesa-dev libglew-dev pkg-config libglu1-mesa-dev freeglut3-dev mesa-common-dev libcairo2-dev xserver-xorg-dev libxext-dev g++-4.8 xvfb
```

Combine and remove duplicates.